explore-analyze/report-and-share/automating-report-generation.md

```diff
@@ -159,3 +165,90 @@ If you experience issues with the deprecated report URLs after you upgrade {{kib
 :::{important}
 In earlier {{kib}} versions, you could use the `&sync` parameter to append to report URLs that held the request open until the document was fully generated. The `&sync` parameter is now unsupported. If you use the `&sync` parameter in Watcher, you must update the parameter.
 :::
+
+## Schedule and share reports [schedule-report-generation]
+
+```{applies_to}
+stack: preview 9.1
+serverless: preview
+```
+
+Save time by setting up a recurring task that automatically generates reports and shares them on a schedule that you choose. 
+
+### Prerequisites [scheduled-reports-reqs]
+
+* To generate PDF and PNG reports, your {{kib}} instance needs a minimum of 2GB of RAM. There is no minimum requirement for CSV reports.
+* To use the scheduled reports feature, your role needs [access to reporting](../../deploy-manage/kibana-reporting-configuration.md#grant-user-access).
+* (Optional) To view and manage other users’ reports and schedules, your role needs `All` privileges for the **Manage Scheduled Reports** feature. You can set this by configuring your role's {{kib}} privileges. If your role doesn't have the **Manage Scheduled Reporting** feature privilege, you can only share reports with yourself. 
+* Sharing reports outside of {{kib}} requires a default preconfigured email connector.
+
+   * **{{ech}} or {{serverless-short}} users**: You do not need to set up a default preconfigured email connector. Kibana provides you with a built-in preconfigured email connector that uses the SMTP protocol to send emails. To view it, go to the **Connectors** page and find the Elastic-Cloud-SMTP connector.
+   * **Self-managed users**: You must set up a default preconfigured email connector to send notifications outside of {{kib}}. To do this:
+     
+     1. Open your `kibana.yml` file.
+     2. Add the `xpack.actions.preconfigured` {{kib}} setting. This setting specifies configuration details for the preconfigured connector that you're defining. 
+     2. Under the `xpack.actions.preconfigured` setting, define the email connector. Refer to [Email connectors](kibana://reference/connectors-kibana/pre-configured-connectors.md#preconfigured-email-configuration) to learn about requirements for different email services and providers.
```

Comment:
There are two step 2s, but they seem to render correctly. I knew you could just use 1. for every step, but didn't know it would render correctly in this instance. TIL!